tests/test_fmi3.py

```diff
@@ -581,6 +580,28 @@ def test_get_input_list(self):
 
         assert expected == list(inputs.keys())
 
+    def test_get_int32_and_64(self):
```

Comment:
Possibly split into 2 different tests; maybe parameterize? Will likely need/want tests for the other int types.

Reply:
Split it into 2 for now, I think we can do the parametrization when we add the other tests later.